### DISCUSSION TITLE
Counts for modes of legacy game types

### DISCUSSION THREAD

**Discussion by mdashlw** [HypixelDev/PublicAPI#492] [General]
It's currently impossible to get the player count for modes of the legacy games, we only have this:
![image](https://user-images.githubusercontent.com/34678047/143727943-710ded03-65c7-47c9-a72e-cd1e86c55f71.png)

This is most likely some weird implementation detail for handling legacy game types, though it'd be great if there was a workaround.

**Comment by phoenixlit** [comment]
Excellent idea, I don't know why this hasn't been added e.g. quake solo, quake teams, arena 1v1, arena 2v2, arena 4v4 counts @ConnorLinfoot
